packages/core/src/elements/Accordion/FeAccordion/FeAccordionContent/index.ts

```diff
@@ -0,0 +1 @@
+export * from './FeAccordionContent';
```

Comment:
- no need folder for each sub components
- just put the component inside FeAccordionContent.tsx, and the FeAccordion/index.ts should export them